prometheus-to-sd/main.go

```diff
@@ -51,6 +51,8 @@ var (
 		"Namespace name of the pod in which monitored component is running.")
 	zoneOverride = flag.String("zone-override", "",
 		"Name of the zone to override the default one (in which component is running).")
+	useNewResources = flag.Bool("use-new-resources", false,
```

Comment:
Please not "new" :-) Can we name it somehow? A year from now, this won't be new anymore, but the flag will stick for a while.

Reply:
Updated